prow/config/default.yaml

```diff
@@ -7,7 +7,6 @@ featureGates:
   GRPCContainerProbe: true
 kubeadmConfigPatches:
   - |
-    apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
I don't fully understand why but I think putting this doesn't work, kind now ignores it. So seems best to leave it off.

Reply:
Moving this change to istio.io, I end up seeing an error:
```
+ kind create cluster --name=istio-testing --config ./prow/config/default.yaml -v4 --retain --image gcr.io/istio-testing/kind-node:v1.27.0 --wait=180s
ERROR: failed to create cluster: unknown apiVersion: 
```
See https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio.io/13152/doc.test.profile_none_istio.io/1653406280368263168/build-log.txt as an example.

Reply:
@howardjohn The istio.io tests only pass if the apiVersion is there. When I removed it in one commit, the tests failed with the above message. I removed that commit, so it was in the yaml, and the tests passed.

Reply:
Huh. I cannot grok why it only works to NOT have it here, but istio.io needs it